openlibrary/templates/books/add.html

```diff
@@ -68,6 +68,12 @@ <h1>$_("Add a book to Open Library")</h1>
                 <div class="label">$_("Since you are not logged in, please satisfy the reCAPTCHA below.")</div>
                 <div class="g-recaptcha" data-sitekey="$:recaptcha.public_key"></div>
             </div>
+        
+        <div class="formElement">
+            <div class="cclicense">
+                $:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="This link to Creative Commons will open in a new window">CC0</a>. Yippee!')
+            </div>
+        </div>
 
         <div class="formElement bottom">
             <div class="input">
```

Comment:
```suggestion
        <div class="formElement bottom">
            <br/>
            <div class="cclicense">
                $:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="This link to Creative Commons will open in a new window">CC0</a>. Yippee!')
            </div>
            <div class="input">
```
Also, remove the `<br/>` found before the "Add" button.